pyprland/plugins/scratchpads/__init__.py

```diff
@@ -5,7 +5,7 @@
 import time
 from dataclasses import dataclass
 from functools import partial
-from typing import cast
+from typing import cast, List
```

Comment:
Which version of Python are you using?
In theory you can skip this import and use the built-in "list"

Reply:
It didn't look in use anyway...

Reply:
Oops, yep, from an older version of Python.